pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
```

Comment:
It kinda works, but deviates slightly from the structure of other command descriptions; the initial verb (open) refers to the command, while the second (type) refers to the user, so the sentence technically has two subjects. But that's a minor point, so if you really prefer it like that, I won't object :) let me know what you think.